fdbrpc/include/fdbrpc/DDSketch.h

```diff
@@ -207,7 +207,7 @@ class DDSketchBase {
 	uint64_t populationSize, zeroPopulationSize; // we need to separately count 0s
 	std::vector<uint64_t> buckets;
 	T minValue, maxValue, sum;
-	void setBucketSize(size_t capacity) { buckets.resize(capacity, 0); }
+	void setBucketSize(int capacity) { buckets.resize(capacity, 0); }
```

Comment:
We should assert that errorGuarantee is not so low that it will cause capacity to overflow.